Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+@Library('fh-pipeline-library')
```

Comment:
Could you replace this with these lines? https://github.com/aerogear/mobile-cli/blob/master/Jenkinsfile#L1..L4

The `#!groovy` is just an editor/IDE hack so that anyone making changes to this file will have a greater chance of their editor recognizing it as a Groovy file, and doing syntax highlighting based on that (if they don't already match on 'Jenkinsfile').

The `_` after the annotation means that we're annotating `_` rather than `import org.feedhenry.Utils` (which can then be removed, because you don't use it here). I think it's cleaner than annotating an import that you don't need so that you get the side-effect of having everything else from the library imported. I originally got this idea from the callout here: https://jenkins.io/doc/book/pipeline/shared-libraries/#using-libraries

Reply:
sure, thanks for that, I'll fix it.